DigitalLearningSolutions.Web/Views/ResetPassword/Index.cshtml

```diff
@@ -0,0 +1 @@
+<h1>Reset Password</h1>

```

Comment:
Super minor, but could we have this "Reset password" to make the capitalisation similar to other pages? Also, whilst we're here, might be worth adding the page title now, so it's not "- DLS" on the tab.

Reply:
Oh, yeah, this page is going to be completely changed in HEEDLS-369 (This is just a placeholder view). So I'll make sure that's done then and there.